translations/fr/info/task_description.html

```diff
@@ -0,0 +1,167 @@
+Décrivons les règles de cette mise en forme :
+
+    <ul class="rules">
+        <li>
+        La vache est toujours la même, seule sa citation change.
+        </li>
+        <li>
+        Des espaces multiples dans une ligne sont remplacés par un seul.
+        </li>
+        <li>
+        La bordure supérieure est constituée de caractères "_" (souligné - "underscore").
+        Elle commence avec un seul espace et se termine avant la colonne de bordure.
+        </li>
+        <li>
+        Chaque ligne de la citation est constituée des parties suivantes :
+        bordure de citation(1), espace(1), ligne(1-39), espace(1), bordure de citation(1).
+        </li>
+        <li>
+        Si le texte fait moins de 40 caractères, il tient dans une seule chaîne de caractères (string).
+        Cette chaîne est entourée par <strong>&lt;&gt;</strong>.
+        </li>
+        <li>
+        Si le texte fait  40 caractères ou plus, il faut le scinder selon les règles suivantes: 
+            <ul>
+                <li>
+                La taille maximale d'un ligne est 39 caractères.
+                Si la ligne trop longue contient un espace, la couper au niveau de l'espace le plus à droite.
+                Cet espace sera alors ôté du texte.
+                Sinon, prendre les 39 premiers caractères.
+                </li>
+                <li>
+                Après le découpage, aligner toutes les lignes sur la gauche, en rajoutant si besoin des espaces au bout des lignes
+                trop courtes, de façon à ce que toutes les lignes fassent la même longueur.
+                    
+                <li>
+                <li>
+                    Bordures de la première ligne: <strong>/\</strong>
+                </li>
+                <li>
+                    Bordures des lignes du milieu: <strong>||</strong>
+                </li>
+                <li>
+                    Bordures de la dernière ligne: <strong>\/</strong></li>
+            </ul>
+        </li>
+        <li>
+        La bordure inférieure est constituée de signes moins "-". Elle a la même longueur que la bordure supérieure.
+        </li>
+        <li>
+        Le programme pour la console "cowsay" a un comportement étrange dans certains cas spéciaux, qui ne seront pas testés ici.</li>
+    </ul>
+</div>
+
+<div>
+    <pre>
+     _________________________________
+    / Dog goes woof                   \
+    | Cat goes meow                   |
+    | Bird goes tweet                 |
+    | And mouse goes squeek           |
+    | Cow goes moo                    |
+    | Duck goes quack                 |
+    \ And the solution will go to you /
+     ---------------------------------
+            \   ^__^
+             \  (oo)\_______
+                (__)\       )\/\
+                    ||----w |
+                    ||     ||
+    </pre>
+    <strong>What does the cow say? (Que dit la vache ?)</strong>
+</div>
+
+<br>
+<p>
+    <strong>Input: </strong> Un texte (string).
```

Comment:
`<strong>Input: </strong>` -> `<strong>Input:</strong>`  no need to have two spaces here.

same for `Output`